contracts/strategies/traderJoe/strategy-joe-avax-mai.sol

```diff
@@ -36,7 +37,7 @@ contract StrategyJoeAvaxMaiLp is StrategyJoeFarmBase {
         //      if so, a new strategy will be deployed.
 
         // Collects Joe tokens
-        IMasterChefJoeV2(masterChefJoeV2).deposit(poolId, 0);
+        IMasterChefJoeV2(masterChefJoeV3).deposit(poolId, 0);
```

Comment:
please update the harvest function accordingly so that it accepts wavax rewards as well.